__tests__/NCN.test.js

```diff
@@ -121,4 +121,43 @@ describe('GET api/article/:articleid', () => {
             expect(body.msg).toBe('bad request')
         })
     });
+});
+
+//task 5
+describe('GET /api/articles', () => {
+    test('GET: 200 should respond with an array of all article objects with expected properties', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            const body = response.body
+            body.forEach((article) => {
```

Comment:
we MUST do a length check of some sort.
If this test was passed an empty array this would cause a false pass.
Please add this and remember this going forward